microcloud/cmd/microcloud/add.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/canonical/microcluster/microcluster"
+	lxdAPI "github.com/lxc/lxd/shared/api"
+	cli "github.com/lxc/lxd/shared/cmd"
+	"github.com/lxc/lxd/shared/logger"
+	"github.com/spf13/cobra"
+
+	"github.com/canonical/microcloud/microcloud/api"
+	"github.com/canonical/microcloud/microcloud/service"
+)
+
+type cmdAdd struct {
+	common *CmdControl
+
+	flagAuto bool
+	flagWipe bool
+}
+
+func (c *cmdAdd) Command() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add",
+		Short: "Begins scanning for new cluster members",
+		RunE:  c.Run,
+	}
+
+	cmd.Flags().BoolVar(&c.flagAuto, "auto", false, "Automatic setup with default configuration")
```

Comment:
Could we call this var flagAutoSetup (and any associated function arguments `autoSetup`) as earlier I saw a function that took an argument called `auto bool` and without a function comment it wasn't clear what that meant in the context of the function's name/signature.

Reply:
I've renamed the flag & associated variables to `autoSetup` and I've went ahead and done the same for `wipe` to `wipeAllDisks`